apollo-router/src/plugins/telemetry/config.rs

```diff
@@ -86,6 +89,130 @@ pub(crate) struct Tracing {
     pub(crate) datadog: Option<tracing::datadog::Config>,
 }
 
+#[derive(Clone, Default, Debug, Deserialize, JsonSchema)]
+#[serde(deny_unknown_fields)]
+pub(crate) struct Logging {
+    /// Log format
+    #[serde(default)]
+    pub(crate) format: LoggingFormat,
+    #[serde(default = "default_display_filename")]
+    pub(crate) display_filename: bool,
+    #[serde(default = "default_display_line_number")]
+    pub(crate) display_line_number: bool,
+    /// Log configuration at supergraph level
+    pub(crate) supergraph: Option<LoggingIncludeAttrs>,
+    /// Log configuration for subgraphs
+    pub(crate) subgraph: Option<SubgraphLogging>,
+}
+
+pub(crate) const fn default_display_filename() -> bool {
+    true
+}
+
+pub(crate) const fn default_display_line_number() -> bool {
+    true
+}
+
+impl Logging {
+    pub(crate) fn get_attributes_to_exclude(&self) -> AttributesToExclude {
```

Comment:
the logic is a bit hard to follow here, could you add some comments explaining what is suppsoed to happen? is `SPECIFIC_ATTRIBUTES` the list of default attributes?

Reply:
Yes, bad namings. Will add more comments :)